ga4gh/datamodel/__init__.py

```diff
@@ -229,6 +256,8 @@ class VariantSetCompoundId(DatasetCompoundId):
     """
     fields = DatasetCompoundId.fields + ['variantSet']
     containerIds = DatasetCompoundId.containerIds + [('variantSetId', 1)]
+    obfuscators = [DatasetCompoundId, None]
```

Comment:
We refer to objects by their ids.  All those ids within our system are obfuscated.  If we want to lookup any item by an id, we need the correct (obfuscated) id.